### PR TITLE
seareport-data feedstock

### PR DESCRIPTION
**Pull request by brey**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by brey** [comment]
@pmav99 Please confirm acceptance as maintainer.

**Comment by pmav99** [comment]
I am willing to be listed as a maintainer.

**Comment by pmav99** [comment]
@conda-forge-admin please rerender

File: recipes/seareport-data/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  sha256: 74e632d09f211318658c5a9e3f7605b5e5d60a5c50d16286ab68382eb9afe57b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10,<4.0
```
Comment (synapticarbors) [inline recipes/seareport-data/meta.yaml:19]: This looks generally good, but could you satisfy the linter by following the docs on handling `noarch: python` recipes:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

You can set a jinja variable for `python_min` specifying 3.10

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/seareport-data/meta.yaml (hunk 1 above)
Comment (brey) [reply recipes/seareport-data/meta.yaml]: Done

**Pull request merged**